excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreator.java

```diff
@@ -323,30 +326,29 @@ public String createErrorMessage(final EventEntityInvalidCrudValidationError err
         return newMessageIfDefinitionExists(error,
             error.getEventACLEntity(),
             def -> String.format(
-                "Invalid CRUD value '%s' in %s tab, case type '%s', event '%s', user "
-                    + "" + "" + "role '%s'",
+                "Invalid CRUD value '%s' in %s tab, case type '%s', event '%s', access profile '%s'",
                 defaultString(def.getString(ColumnName.CRUD)),
                 def.getSheetName(),
                 error.getAuthorisationEventValidationContext().getCaseReference(),
                 error.getAuthorisationEventValidationContext().getEventReference(),
-                defaultString(def.getString(ColumnName.USER_ROLE))));
+                defaultString(def.getString(ColumnName.ACCESS_PROFILE))));
     }
 
     @Override
     public String createErrorMessage(final EventEntityInvalidUserRoleValidationError error) {
         return newMessageIfDefinitionExists(error,
             error.getEventACLEntity(),
-            def -> CaseRoleServiceImpl.isCaseRole(defaultString(def.getString(ColumnName.USER_ROLE)))
+            def -> CaseRoleServiceImpl.isCaseRole(defaultString(def.getString(ColumnName.ACCESS_PROFILE)))
                 ? String.format(
                 "Invalid case role '%s' in %s tab, case type '%s', event '%s'. "
                     + DEFINED_IN_CASE_ROLES_SHEET_VERIFICATION_MESSAGE,
-                defaultString(def.getString(ColumnName.USER_ROLE)),
+                defaultString(def.getString(ColumnName.ACCESS_PROFILE)),
                 def.getSheetName(),
                 error.getAuthorisationEventValidationContext().getCaseReference(),
                 error.getAuthorisationEventValidationContext().getEventReference()) :
                 String.format(
-                    "Invalid idam role '%s' in %s tab, case type '%s', event '%s', crud '%s'",
-                    defaultString(def.getString(ColumnName.USER_ROLE)),
+                    "Invalid IDAM access profile '%s' in %s tab, case type '%s', event '%s', crud '%s'",
```

Comment:
Remove the word IDAM

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f